sdk/core/src/http_client.rs

```diff
@@ -92,7 +96,9 @@ impl HttpClient for hyper::Client<HttpsConnector<hyper::client::HttpConnector>>
     }
 }
 
-#[async_trait]
+#[cfg(feature = "enable_reqwest")]
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
```

Comment:
I didn't know about `cfg_attr`, but thanks to https://github.com/dtolnay/async-trait/issues/159#issuecomment-819241829, it works perfectly. `Send` is only disabled for wasm builds. Everything else works as before.